pangea-sdk/service/vault/secret.go

```diff
@@ -0,0 +1,30 @@
+package vault
+
+type SecretStoreRequest struct {
+	CommonStoreRequest
+	Secret string `json:"secret"`
+}
+
+type SecretStoreResult struct {
+	CommonStoreResult
+	Secret string `json:"secret"`
+}
+
+type SecretRotateRequest struct {
+	CommonRotateRequest
+	Secret string `json:"secret"`
+}
+
+type SecretRotateResult struct {
+	CommonRotateResult
+	Secret string `json:"secret"`
+}
+
+type PangeaTokenRotateRequest struct {
+	CommonRotateRequest
+}
```

Comment:
I think we should provide rotationgraceperiod here

Reply:
fixed